py-polars/polars/io/database/functions.py

```diff
@@ -76,9 +76,10 @@ def read_database(  # noqa: D417
     connection
         An instantiated connection (or cursor/client object) that the query can be
         executed against. Can also pass a valid ODBC connection string, identified as
-        such if it contains the string "Driver=", in which case the `arrow-odbc`
+        such if it contains the string "Driver={...}", in which case the `arrow-odbc`
         package will be used to establish the connection and return Arrow-native data
-        to Polars.
+        to Polars. Async driver connections are also supported, though this is currently
+        considered experimental.
```

Comment:
There's no actual user-facing API change here, so I might tweak the warning to indicate that it may not be fully baked yet, rather than it might change 🤔  (because any changes would be invisible to the user; it's more like _"your mileage may vary"_, rather than you may have to update the calling code).

Reply:
Sure it's fine to tweak the message a bit, but there really must be a Sphinx warning admonition here with `Asynchronous connection support is considered **unstable**.` (unstable in bold). I really want to be very consistent with this across the docs.

Reply:
Ahh, forgot that (unintentional); one sec, will fix.

Reply:
I added the **bold** part :grimacing:

Reply:
My attention is a bit split; still at work. At 20:00 😅😝